lib/SILGen/SILGenThunk.cpp

```diff
@@ -70,47 +70,6 @@ SILFunction *SILGenModule::getDynamicThunk(SILDeclRef constant,
   return F;
 }
 
-// SWIFT_ENABLE_TENSORFLOW
-SILFunction *
-SILGenModule::getOrCreateAutoDiffDerivativeForwardingThunk(
```

Comment:
I believe as is, `getOrCreateAutoDiffDerivativeForwardingThunk` created a more efficient thunk, as it never reabstracts the returned linear map while `getOrCreateCustomDerivativeThunk` always does. Not a big deal!